src/storage/src/hummock/sstable/multi_builder.rs

```diff
@@ -228,6 +229,15 @@ where
                     // default
                     self.largest_vnode_in_current_partition = VirtualNode::MAX.to_index();
                 }
+            } else {
+                self.last_table_id = user_key.table_id.table_id;
+                self.split_weight_by_vnode = 0;
+                self.largest_vnode_in_current_partition = VirtualNode::MAX.to_index();
+                if let Some(builder) = self.current_builder.as_ref()
+                    && builder.approximate_len() > MIN_SST_SIZE
```

Comment:
The change here means that we will split by table as long as the prev table as accumulated 4MB data if no vnode partition is set for the prev table and the cur table. There are two things that struggle me:
1. Why do we choose 4M as the threshold? Any assumption or background here?
2. If either prev or cur table has vnode partition set, we will split the SST regardless of whether 4M has reached. That meas it is possible that the previous table can be split when its size > 4MB. This is kind of strange because a stronger setting (vnode partition set) leads to a weaker strategy (more data accumulated).

Reply:
Because a large split threhold may cause we can not calculate the size of each table accurately in meta-node.

Reply:
A bad news is that we can not configure it in meta-node for flush job.

Reply:
I add a new configuration for flush job